EIPS/eip-FOCIL.md

```diff
@@ -26,10 +26,79 @@ FOCIL is a simple committee-based design improving upon previous IL mechanisms o
 
 ## Specification
 
+### Constants
+
+| Name | Value |
+| - | - |
+| `MAX_TRANSACTIONS_PER_INCLUSION_LIST` | `2**4 = 16` |
+| `MAX_GAS_PER_INCLUSION_LIST` | `2**21` |
+| `MAX_TRANSACTIONS_PER_INCLUSION_LIST_AGGREGATE` | `2 * MAX_TRANSACTIONS_PER_INCLUSION_LIST = 32` |
+| `MAX_GAS_PER_INCLUSION_LIST_AGGREGATE` | `2 * MAX_GAS_PER_INCLUSION_LIST = 2**22` |
+| `IL_COMMITTEE_SIZE` | `256` |
+
+### Reference Objects
+
+```
+class InclusionListEntry(Container):
+    from_address: ExecutionAddress
+    gas_limit: uint64
+```
+
+```
+class inclusionList(Container)
+    slot: Slot
+    proposer_index: ValidatorIndex
+    entries: List[InclusionListEntry, MAX_TRANSACTIONS_PER_INCLUSION_LIST]
+```
+
+```
+class InclusionListAggregatedEntry(Container):
+    from_address: ExecutionAddress
+    gas_limit: uint64
+    bitlist: List[byte, IL_COMMITTEE_SIZE]
+```
+
+```
+class InclusionListAggregated(Container):
+    slot: Slot
+    proposer_index: ValidatorIndex
+    message: List[InclusionListAggregatedEntry, MAX_TRANSACTIONS_PER_INCLUSION_LIST_AGGREGATE]
+```
+
 ### Consensus layer
 
 #### High-level overview
 
+TBA (Same as timeline)
+
+#### Specific changes
+
+**Beacon chain state transition spec:**
+
+- ***New** `inclusion_list` object:* Introduce a new `inclusion_list` for the IL committee to submit and nodes to process.
+- ***Modified** `BeaconBlockBody`:* Modified to include the aggregated inclusion list for that slot.
+- ***Modified** `process_execution_payload` function:* Update this process to include checks for the inclusion list evaluation.
+
+**Beacon chain P2P spec:**
+
+- ***New** gossipnet and validation rules for inclusion list:* Define new rules for handling the inclusion list in the gossip network and validation.
+- ***New** RPC request and response network for inclusion list:* Establish a new network for sending and receiving inclusion lists.
```

Comment:
we should add a new line for inclusion list aggregate as well

Reply:
Got it. I am assuming we need to duplicate both the above points for aggregated IL.